### ISSUE TITLE
Korrekt bruk av role=alert i Driftsmeldinger

### ISSUE THREAD

**Issue by PerOlavM** [navikt/nav-dekoratoren_legacy#1491]
https://nav-it.slack.com/archives/CAFRFDJMN/p1695119317046909
